src/backend/TrafficCourts/Citizen.Service/Models/Search/TicketSearchResult.cs

```diff
@@ -1,15 +1,51 @@
-﻿using System.Diagnostics.CodeAnalysis;
+﻿using Swashbuckle.AspNetCore.Annotations;
+using System.Diagnostics.CodeAnalysis;
 
 namespace TrafficCourts.Citizen.Service.Models.Search
 {
     /// <summary>
-    /// Represents a violation ticket that is returned from search requests
+    /// Obsolete: Represents a violation ticket that is returned from search requests
     /// </summary>
+    [Obsolete]
     public class TicketSearchResult
     {
-        public TicketSearchResult()
+        public TicketSearchResult(Tickets.ViolationTicket violationTicket)
```

Comment:
Just used to map the new `ViolationTicket` to the obsolete version until the front end can be updated to use the new models.